helm/rayyan-store-proxy/templates/secrets/rayyan-store-proxy-redis.yaml

```
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: rayyan-store-proxy-redis
+type: Opaque
+stringData:
+  REDIS_URL: {{ .Values.secrets.RAYYAN_STORE_PROXY_REDIS_URL }}
```

Now you can de-scope the secret name (no more monolith chart):
```suggestion
  REDIS_URL: {{ .Values.secrets.REDIS_URL }}
```

---

Done.